pkg/network/network.go

```diff
@@ -0,0 +1,146 @@
+package network
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/skupperproject/skupper/api/types"
+	"strconv"
+	"strings"
+)
+
+type SkupperStatus struct {
+	VanStatus *types.VanStatusInfo
+}
+
+func (s *SkupperStatus) GetServiceSitesMap() map[string][]types.SiteStatusInfo {
+
+	mapServiceSites := make(map[string][]types.SiteStatusInfo)
+
+	for _, site := range s.VanStatus.SiteStatus {
+
+		for _, listener := range site.RouterStatus[0].Listeners {
```

Comment:
I have noticed that when a podman site is linked the `RouterStatus` slice is empty.
Maybe you could ignore sites with an empty RouterStatus slice for now.

Reply:
Thanks! solved in 3371b9e46ea653a9f283ff1dd6c3bcff3e0ef4ae